lib/vmpooler/api.rb

```diff
@@ -2,51 +2,62 @@
 
 module Vmpooler
   class API < Sinatra::Base
-    def initialize
-      super
+    # Load API components
+    %w[helpers dashboard reroute v1 request_logger].each do |lib|
+      begin
+        require "vmpooler/api/#{lib}"
+      end
+    end
+    # Load dashboard components
+    begin
+      require 'vmpooler/dashboard'
     end
```

Comment:
With the rescues removed the begin/end blocks should also go.

```suggestion
    %w[helpers dashboard reroute v1 request_logger].each do |lib|
      require "vmpooler/api/#{lib}"
    end
    # Load dashboard components
    require 'vmpooler/dashboard'
```

Reply:
👍